addons/web/static/src/views/view_dialogs/form_view_dialog.js

```diff
@@ -115,6 +115,7 @@ FormViewDialog.props = {
     preventCreate: { type: Boolean, optional: true },
     preventEdit: { type: Boolean, optional: true },
     isToMany: { type: Boolean, optional: true },
+    size: { type: String, optional: true },
```

Comment:
I see that the `size` props of the Dialog component is defined with`size: { type: String, optional: true, validate: (s) => ["sm", "md", "lg", "xl"].includes(s) },` and also sets a default to `lg` via `defaultProps`. Should not we do the same here? Even if I guess the validation is done in the end by the Dialog sub component but making it sooner might be better (although I guess it would be done twice...)? But mainly it would prevent code duplication and make it clear that this is the Dialog props you allow the control here. Something like: `size: Dialog.props.size`. Maybe confirm with the framework team if needed.

Reply:
Well, since the `PagePropertiesDialog` is added using dialog service (using a `DialogContainer` component which contains dialogs as sub-components with their dynamic props...), I'm not sure the "DEV mode props validation" will be done in this case (to be confirmed by @odoo/rd-framework-js) but I think the `size: Dialog.props.size description` description is better anyway :+1: